contrib/ubase/Makefile

```diff
@@ -0,0 +1,133 @@
+TOPDIR = $(realpath ../..)
+
+
+HDR = \
+	arg.h \
+	compat.h \
+	crypt.h \
+	fs.h \
+	md5.h \
+	queue.h \
+	sha1.h \
+	sha224.h \
+	sha256.h \
+	sha384.h \
+	sha512.h \
+	sha512-224.h \
+	sha512-256.h \
+	text.h \
+	utf.h \
+	util.h
+
+LIBUTILSRC = \
+	libutil/agetcwd.c        \
+	libutil/agetline.c       \
+	libutil/apathmax.c       \
+	libutil/concat.c         \
+	libutil/ealloc.c         \
+	libutil/eprintf.c        \
+	libutil/estrtol.c        \
+	libutil/estrtoul.c       \
+	libutil/explicit_bzero.c \
+	libutil/passwd.c         \
+	libutil/proc.c           \
+	libutil/putword.c        \
+	libutil/recurse.c        \
+	libutil/strlcat.c        \
+	libutil/strlcpy.c        \
+	libutil/strtonum.c       \
+	libutil/tty.c
+
+BIN = \
+	clear             \
+	getty							\
+	id                \
+	login             \
+	nologin           \
+	pagesize          \
+	respawn           \
+	stat              \
+	su                \
+	truncate          \
+	watch
+
+BIN_DONT_WORK = \
+	blkdiscard        \
+	chvt              \
+	ctrlaltdel        \
+	dd                \
+	df                \
+	dmesg             \
+	eject             \
+	fallocate         \
+	free              \
+	freeramdisk       \
+	fsfreeze          \
+	halt              \
+	hwclock           \
+	insmod            \
+	killall5          \
```

Comment:
What stops us from porting `killall5` tool?

Reply:
`killall5` implementation in `ubase` is dependent on `proc` pseudo filesystem (https://git.suckless.org/ubase/file/killall5.c.html#l89)